core/templates/pages/exploration-player-page/services/audio-preloader.service.ts

```diff
@@ -93,11 +100,18 @@ export class AudioPreloaderService {
   private getAudioFilenamesInBfsOrder(sourceStateName: string): string[] {
     const languageCode = (
       this.audioTranslationLanguageService.getCurrentAudioLanguageCode());
-    const allVoiceovers = this.exploration.getAllVoiceovers(languageCode);
-    const bfsTraversalOfStates = (
-      this.computeGraphService.computeBfsTraversalOfStates(
-        this.exploration.getInitialState().name, this.exploration.getStates(),
-        sourceStateName));
+    if (languageCode === null) {
+      return [];
+    }
```

Comment:
Please add an explanatory comment for this.

Reply:
Done.